src/app/shared/model/audio-version-template.model.ts

```diff
@@ -3,37 +3,45 @@ import 'rxjs/add/observable/of';
 import 'rxjs/add/operator/map';
 import { HalDoc } from '../../core';
 import { BaseModel } from 'ngx-prx-styleguide';
-import { REQUIRED, LENGTH, VERSION_LENGTH } from './invalid';
+import { REQUIRED, LENGTH, IN, VERSION_LENGTH } from './invalid';
 import { AudioFileTemplateModel } from './audio-file-template.model';
 
 export class AudioVersionTemplateModel extends BaseModel {
 
   public id: number;
   public label: string = null;
+  public contentType = 'audio/mpeg';
   public lengthMinimum: number = null;
   public lengthMaximum: number = null;
   public fileTemplates: AudioFileTemplateModel[];
+  private newIndex: number = null;
 
-  SETABLE = ['label', 'lengthMinimum', 'lengthMaximum'];
+  SETABLE = ['label', 'contentType', 'lengthMinimum', 'lengthMaximum'];
 
   VALIDATORS = {
     label: [REQUIRED(), LENGTH(1, 255)],
+    contentType: [REQUIRED(), IN(['audio/mpeg', 'audio/mp2', 'video/mpeg', 'video/x-mpeg'])],
```

Comment:
This is nitpicky, but since they are used more than once, I'm wishing these strings were constants.